File: examples/docs-code-snippets/src/main/java/examples/java/recipes/RecipeWithEventReceivePeriod.java
package examples.java.recipes;

import com.ing.baker.recipe.javadsl.Recipe;

import java.time.Duration;

public class RecipeWithEventReceivePeriod {

    public final static Recipe recipe = new Recipe("example")
        .withEventReceivePeriod(Duration.ofHours(5));
}

File: examples/docs-code-snippets/src/main/kotlin/examples/kotlin/recipes/RecipeWithEventReceivePeriod.kt
package examples.kotlin.recipes

import com.ing.baker.recipe.kotlindsl.ExperimentalDsl
import com.ing.baker.recipe.kotlindsl.recipe
import kotlin.time.Duration.Companion.hours

@ExperimentalDsl
object RecipeWithEventReceivePeriod {
    val recipe = recipe(name = "example") {
        eventReceivePeriod = 5.hours
    }
}